csharp/ql/src/Security Features/CWE-937/VulnerablePackage.ql

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name Using a package with a known vulnerability
+ * @description Using a package with a known vulnerability is a security risk.
+ *              Upgrade the package to a version that does not contain the vulnerability.
+ * @kind problem
+ * @problem.severity error
+ * @precision high
+ * @id cs/use-of-vulnerable-package
+ * @tags security
+ *       external/cwe/cwe-937
+ */
+
+import csharp
+import Vulnerabilities
+
+from Vulnerability vuln, VulnerablePackage package
+where vuln = package.getVulnerability()
+select package, "Package " + package + " has vulnerability $@, and should be upgraded to version " + package.getFixedVersion() + ".",
```

Comment:
add `'` around package

Reply:
Have you checked that the links to `vuln.getUrl()` render correctly in both QL4E and on LGTM.com?

Reply:
@calumgrant : Did you check this?

Reply:
If the above does not work, I think something like this should:
```
"Package " + package + " has vulnerability [[\"" + vuln + "\"|\"" +vuln.getUrl()+ "\"]], and should be upgraded to version " + package.getFixedVersion() + "."
```

Reply:
If the above does not work, I think something like this should:
```
"Package " + package + " has vulnerability [[\"" + vuln + "\"|\"" +vuln.getUrl()+ "\"]], and should be upgraded to version " + package.getFixedVersion() + "."
```

Reply:
If the above does not work, I think something like this should:
```
"Package " + package + " has vulnerability [[\"" + vuln + "\"|\"" +vuln.getUrl()+ "\"]], and should be upgraded to version " + package.getFixedVersion() + "."
```